ichnaea/data/area.py

```diff
@@ -164,7 +157,7 @@ def update_area(self, session, areaid):
         if cell_last_seen:
             last_seen = max(cell_last_seen)
 
-        if area is None:
+        if area_is_new:
```

Comment:
I preferred to use ``area_exists``, but ``area_is_new`` makes this existing statement more natural, and keeps it a one-line change.